### ISSUE TITLE
Move parent if children is moved

### ISSUE THREAD

**Issue by Zilvinaskli** [clientIO/joint#1260]
I'm struggling to move the parent together with children if embedded children are moved. 

Firstly I've tried something like that: 

```
this.graph.on('change:position', (cell, collection, opt) => {
        const parent = cell.getParentCell();

        if (parent && parent.attributes.type === 'standard.Combo') {
          parent.translate(opt.tx, opt.ty);
}
```
but this gives me infinite loop as translating parent translates children as well. 

I currently have parent.position(cell.attributes.position.x + opt.tx, cell.attributes.position.y + opt.ty, { deep: false }); but this is not smooth.

I also found this perfect solution: [https://stackoverflow.com/a/45440557/5943087](https://stackoverflow.com/a/45440557/5943087)

however, it does not work with versions 2.1 and above. I've tried overwriting pointermove event with the event of 2.0 but then I was not able to move anything on the graph. Is there a way to make this solution work?

**Comment by kumilingus** [comment]
Hi, you probably looking for the interactivity option `stopDelegation` (https://resources.jointjs.com/docs/jointjs/v3.1/joint.html#dia.Paper.prototype.options.interactive) (Not available in v2.1 though).

Instead of `translate()` method please use `parent.set('position',  { x: parent.attributes.position.x + opt.tx, y: parent.attributes.position.y + opt.ty })`.

**Comment by Zilvinaskli** [comment]
Brilliant, stopDelegation was what I needed for the children.
